collectors/python.d.plugin/python.d.conf

```diff
@@ -41,6 +41,7 @@ chrony: no
 # dockerd: yes
 # dovecot: yes
 # elasticsearch: yes
+energi: no
```

Comment:
If there are no jobs defined for the module (and the included config does not define any), it won't be run by default, even if it would be according to python.d.conf.  Given this, the standard for modules that require local configuration is to list them as enabled by default in python.d.conf (so just like the surrounding lines) so that users only have to add jobs to the configuration for the module itself to enable it.

Reply:
OK